Sources/eppo/Poller.swift

```diff
@@ -0,0 +1,198 @@
+import Foundation
+
+public enum PollerConstants {
+    public static let DEFAULT_POLL_INTERVAL_MS = 300_000 // 5 minutes in milliseconds
+    public static let DEFAULT_JITTER_INTERVAL_RATIO = 10 // 10% of poll interval
+    public static let DEFAULT_MAX_POLL_RETRIES = 7
+}
+
+class PollerLogger {
```

Comment:
Does the swift builtin `Logger` type work for logging purposes here?

Reply:
I should have checked. Swift does have a built in logger! I like this idea, but I would prefer to save implementing it for a separate PR so I can switch out all of our `print` statements, not just the ones relevant for this work